src/Simplex/Messaging/Server.hs

```diff
@@ -795,6 +800,15 @@ smpServer started cfg@ServerConfig {transports, transportConfig = tCfg} attachHT
                     logError "Unauthorized control port command"
                     hPutStrLn h "AUTH"
 
+    majorGCThread_ :: ServerConfig -> [M ()]
+    majorGCThread_ ServerConfig {majorGCInterval = Just interval} = [majorGCThread $ interval * 1000000]
+      where
+        majorGCThread int = forever $ do
+          threadDelay int
+          logInfo "Starting major GC"
```

Comment:
More like `logDebug`. The message gives even less information than the port bindings, is inactionable yet periodic.